examples/reference/chat/PanelCallbackHandler.ipynb

```diff
@@ -129,7 +129,8 @@
     "\n",
     "async def callback(contents, user, instance):\n",
     "    callback_handler = pn.chat.langchain.PanelCallbackHandler(instance)\n",
-    "    await agent.arun(contents, callbacks=[callback_handler])\n",
+    "    response = await agent.arun(contents, callbacks=[callback_handler])\n",
```

Comment:
Can you confirm this works for you?

I crash into a NotImplemented error when using async with ddg.

Reply:
Could you be more specific? How can I reproduce this?

Reply:
Try running this example and see if it successfully runs.

Reply:
I can reproduce.

Reply:
Seems like an issue with the DuckDuckGo tool rather than with our code. Just need to change the example to use `agent.run`

Reply:
Fixed.